mp-jwt/src/main/java/org/apache/tomee/microprofile/jwt/config/JWTAuthConfiguration.java

```diff
@@ -29,47 +29,47 @@
 /**
  * The public key and expected issuer needed to validate a token.
  */
-public class JWTAuthContextInfo {
-    private static final Logger logger = Logger.getLogger(JWTAuthContextInfo.class.getName());
+public class JWTAuthConfiguration {
+    private static final Logger logger = Logger.getLogger(JWTAuthConfiguration.class.getName());
     public static final String DEFAULT_KEY = "DEFAULT";
 
-    private Map<String, Key> signerKeys;
-    private String issuedBy;
+    private Map<String, Key> publicKeys;
```

Comment:
Not so sure about this one.
Key is the base class and can be a public key or a secret key.

What about signingKey?
It would be valid for either symmetric or asymmetric key.

Reply:
Changed to "verificationkey" most places